fuzzing/cc_deps.bzl

```diff
@@ -17,6 +17,50 @@
 
 load("@rules_cc//cc:defs.bzl", "cc_test")
 
+def _cc_fuzz_run_impl(ctx):
+    # Generate a script to launcher the fuzzing test
+    script = ctx.actions.declare_file("%s" % ctx.label.name)
+
+    # This command is absolutely not what we are looking for,
+    # but without timeout here, the bazel run never exists.
+    # What is wrong here?
+    script_template = """#!/bin/bash
+timeout 10 {launcher} {target}
+"""
+
+    script_content = script_template.format(
+        launcher =
+            ctx.attr._launcher[DefaultInfo].files_to_run.executable.short_path,
+        target =
+            ctx.attr.dep[DefaultInfo].files_to_run.executable.short_path,
+    )
+    ctx.actions.write(script, script_content, is_executable = True)
+
+    # Merge the two dependencies
+    runfiles = ctx.attr._launcher[DefaultInfo].default_runfiles
+    runfiles = runfiles.merge(ctx.attr.dep[DefaultInfo].default_runfiles)
+
+    return [DefaultInfo(executable = script, runfiles = runfiles)]
+
+# What should the argument be? How to pass timeout_secs?
+cc_fuzz_run = rule(
+    implementation = _cc_fuzz_run_impl,
+    attrs = {
+        # The launcher script to start fuzzing test
+        "_launcher": attr.label(
+            default = Label("//fuzzing/tools:launcher"),
+            executable = True,
+            cfg = "host",
+        ),
+        # The _fuzz_test executable to run
+        "dep": attr.label(
+            executable = True,
+            cfg = "host",
```

Comment:
Should this be "target" instead? I guess we want the `asan-libfuzzer` configuration and all other build settings to apply on this one.